assets/js/app/llms-forms.js

```diff
@@ -378,12 +377,41 @@ LLMS.Forms = {
 			this.$states.html( '<option>&nbsp</option>' );
 			this.disable_field( this.$states );
 		} else {
+			var placeholder = this.get_state_placeholder( this.$states, country_code );
 			this.enable_field( this.$states );
 			this.$states.html( opts );
+			if ( '' !== placeholder ) {
+				this.set_state_placeholder( this.$states, placeholder );
+			}
 		}
 
 	},
 
+    /**
+	 * Get the state placeholder and/or {REGION}
+	 *
+	 * @since [version]
+	 *
+	 * @param {String} placeholder Currently selected state is the placeholder text.
+	 * @return {void}
```

Comment:
```suggestion
	 * @return {string} The placeholder text.
```

Reply:
Please review documentation on how to properly document via jsdoc

parameters: https://jsdoc.app/tags-param.html
return: https://jsdoc.app/tags-returns.html

in this case we need the parameters to have the type, name, and description while the return should document the type and description.

You also have *too many* parameters documented (3) while the function only has two parameters.